packages/editor/src/client/locales/en.json

```diff
@@ -56,6 +56,15 @@
         "emptyLead": "Leave empty to use original lead.",
         "emptyPreTitle": "Leave empty to use original pre-title",
         "emptyTitle": "Leave empty to use original title",
+        "enterImage": "Please add an image",
+        "enterTitle": "Please enter a title",
+        "enterSeoTitle": "Please enter a seo title",
+        "enterSocialMediaDescription": "Please enter a description for social media",
+        "enterSocialMediaAuthors": "Please add an author for social media",
+        "enterSocialMediaTitle": "Please enter a title for social media",
+        "enterSocialMediaImage": "Please enter an image for social media",
```

Comment:
@camilleory How about "Please add an image for social media"

Reply:
Done